### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by DeveloperJosh** [DeveloperJosh/Fish-Mail#16] [Ideas]
Hello, if you have any ideas/commands you want added to the bot let me know here and they will probably get added.

**Comment by Nirlep5252** [comment]
I have a few:

- Customization for ticket open/close messages.
- Ticket logs which is like modlogs but for when users open tickets.
- A per guild ticket template system.
- And maybe change all the current commands from normal messages to embeds for sex purposes.
